shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -28,62 +29,141 @@
 import io.flutter.view.FlutterMain;
 
 /**
- * {@code Activity} which displays a {@link io.flutter.view.FlutterView} that takes up all
- * available space.
+ * {@code Activity} which displays a {@link FlutterFragment} that takes up all available space.
  *
  * {@link FlutterActivity} is the simplest and most direct way to integrate Flutter within an
  * Android app.
- * TODO(mattcarroll): how exactly does FlutterActivity need to be leveraged to display Flutter content?
  *
- * If Flutter is needed in a location that cannot use an {@code Activity>}, consider using
+ * By default, {@link FlutterActivity} configures itself to do the following:
+ *  - no splash screen
+ *  - asks {@link FlutterMain#findAppBundlePath(Context)} for the path to the Dart app bundle
```

Comment:
uber-nit: Here and below `s/asks/ask/`, `s/uses/use/` to correspond to the tense of `to do` above.